build/release.mk

```diff
@@ -53,13 +65,26 @@ ifeq ($(UPLOAD_PACKAGES), true)
 	@bash $(CURDIR)/build/upload_artifacts_gh.sh $(REPO_FULL_NAME)
 endif
 	@echo "===> $(INTEGRATION) === [release/publish] publishing manifests"
-	@$(GORELEASER_BIN) build --config $(CURDIR)/.goreleaser.yml --skip-validate --snapshot --rm-dist
+	@$(GORELEASER_BIN) build --config $(CURDIR)/.goreleaser.yml --skip=validate --snapshot --clean
 
+.PHONY : release/publish-fips
+release/publish-fips:
+ifeq ($(UPLOAD_PACKAGES), true)
```

Comment:
UPLOAD_PACKAGES could be safely change to PRE-RELEASE making everything clearer?

Reply:
done